components/SocialButtons.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Link, Box } from 'ooni-components'
+import { Text } from 'rebass'
+import { TiSocialFacebook, TiSocialTwitter } from 'react-icons/lib/ti'
+
+export default function SocialButtons({ url, fontSize, color }){
+  return(
+    <>
+      <Box px={1}>
+        <Link color={color} target="_blank" href={`https://www.facebook.com/sharer/sharer.php?u=https://explorer.ooni.org/${url}`}>
+            <TiSocialFacebook size={fontSize}/>
+        </Link>
+      </Box>
+      <Box px={1}>
+        <Link color={color} target="_blank" href={`https://twitter.com/intent/tweet?url=https://explorer.ooni.org/${url}`}>
```

Comment:
TODO add support for boilerplate text inside of the target.

Reply:
Please note that this is possible only for Twitter. Facebook used to allow this earlier, but now they changed the behavior to pull the contents from the "og:*" meta tags.
 https://developers.facebook.com/support/bugs/357750474364812/?comment_id=286474311528313

https://developer.twitter.com/en/docs/twitter-for-websites/tweet-button/guides/web-intent

Reply:
I've added text for twitter which says: 'Data from OONI'